### DISCUSSION TITLE
Can't Connect Between Jellyfin/Jellyseerr via Docker Network

### DISCUSSION THREAD

**Discussion by ghost** [seerr-team/seerr#426] [Q&A] [answered]
I get "Invalid User Name/Password" errors connecting to Jellyfin via local docker IP. My password is correct as I can connect fine via WAN hostname, but for some reason I can't connect at all locally. I use http://172.22.0.3/jellyfin to connect since my jellyfin has a base URL set.

I can ping from Jellyseerr to Jellyfin, but wget doesn't work. My log in attempts don't even appear in the Jellyfin logs, and the Jellyseerr logs just have a generic "Failed login attempt from user with incorrect Jellyfin credentials" entry even with debug logs on.

There definitely seems to be something wrong with the connection between containers and since wget doesn't work, maybe it's a port issue. But I don't have any idea how to fix it. I'm using the hotio jellyfin package, maybe that has something to do with it?

[My compose files](https://pastebin.com/6G5Hv1t9)
[My docker network](https://pastebin.com/3LwjCz39)

**Comment by fallenbagel** [comment]
Can you try
```
docker exec -it jellyseerr sh
```
Then
```
apk update && apk add curl && curl http://172.22.0.3/jellyfin
```

Would prefer if you can hop on discord so we can troubleshoot easier with follow ups

**Comment by ghost** [comment, marked as answer]
Needed to add :8096 to http://172.22.0.3:8096/jellyfin
